App.tsx

```diff
@@ -53,8 +54,10 @@ const App: FunctionComponent = () => {
                   <ExposureProvider>
```

Comment:
For context sharing, noting that we talked about cleaning up this Context Mayhem because it's getting out of hand.